Dependencies/d3dx.ini

```diff
@@ -768,10 +843,10 @@ ps-t100 = stereo2mono bb
 ; ps-t100 = stereo2mono ps-t100
 ; Some games such as The Evil Within and Akiba's Trip constantly unbind
 ; StereoParams & IniParams, so we may need to explicitly rebind them now:
-;vs-t125 = StereoParams
-;ps-t125 = StereoParams
-;vs-t120 = IniParams
-;ps-t120 = IniParams
+vs-t125 = StereoParams
```

Comment:
These need to be commented out for top of tree. Might be from the original file you started with, this is a recent change.

I'll put these back when I edit the file.